src/screens/Wallet.js

```diff
@@ -131,29 +128,11 @@ class Wallet extends React.Component {
    * @return {boolean} If should show administrative tab
    */
   shouldShowAdministrativeTab = () => {
-    const walletData = hathorLib.wallet.getWalletData();
-
-    for (const tx_id in this.props.historyTransactions) {
-      const tx = this.props.historyTransactions[tx_id];
-      for (const output of tx.outputs) {
-        // This output is not mine
-        if (!hathorLib.wallet.isAddressMine(output.decoded.address, walletData)) {
-          continue;
-        }
-
-        // This token is not the one of this screen
-        if (output.token !== this.props.selectedToken) {
-          continue;
-        }
-
-        if (hathorLib.wallet.isMintOutput(output) || hathorLib.wallet.isMeltOutput(output)) {
-          return true;
-        }
-
-      }
-    }
+    return this.props.wallet.getMintAuthority(this.props.selectedToken, { skipSpent: false }) || this.props.wallet.getMeltAuthority(this.props.selectedToken, { skipSpent: false })
```

Comment:
I feel this is hard to read. Maybe do something like:

```js
if (condA) {
  return true;
}
if (condB) {
  return true;
}
return false;
```

What do you think?

Reply:
Done